layouts/partials/footer.html

```diff
@@ -3,5 +3,26 @@
 </div>
 </main>
 {{ partial "side-nav.html" . }}
+{{ if isset .Site.Params "darktheme" }}
+<script type="text/javascript">
+    function setTheme(themeName) {
+        document.body.className = themeName;
+    }
+    
+    function selectTheme() {
+        const prefersDarkTheme = window.matchMedia('(prefers-color-scheme: dark)');
```

Comment:
I think we could cut down on some code here with a ternary considering `prefers-color-scheme` is binary, light or dark i.e:

```js
window.matchMedia('(prefers-color-scheme: dark)').matches ? setTheme(...dark) : setTheme(...light)
```

Reply:
Not necessary obviously, works great as is :)